### DISCUSSION TITLE
增加翻译引擎标识显示

### DISCUSSION THREAD

**Discussion by MacroBorn** [YiiGuxing/TranslationPlugin#3699] [Ideas]
建议增加翻译引擎Logo或名称显示，这样当翻译时可以快速知道当前用了哪个引擎进行翻译的，如下
![image](https://github.com/YiiGuxing/TranslationPlugin/assets/22791550/18416f36-4345-4d6c-8e51-0b95df4032b1)
至于显示位置上下左右角落都行

**Comment by YiiGuxing** [comment]
在IDE右下角的状态栏上已经有翻译引擎Logo了的：
![image](https://github.com/YiiGuxing/TranslationPlugin/assets/10737066/04c4c45c-4d34-425f-a70d-697ec8c92413)

翻译面板上已经没有位置放Logo了。。。。

**Reply by MacroBorn** [reply]
那就ok了
